gf-webapp-admin/config-overrides.js

```diff
@@ -1,6 +1,7 @@
-import * as path from "path";
+//eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require("path");
 
-export default{
```

Comment:
Let's move from react-app-rewired to craco, it's more modern.

Reply:
Add comment with explanation